### ISSUE TITLE
Configuring OAuth2 is based on depreciated azure interface

### ISSUE THREAD

**Issue by ddt3** [hevelius/files_external_onedrive#10]
Azure app registrations have changed, which makes it hard to follow the given instructions:

- "Name Your App and then click Create App"
Should include the choice: "Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)"

- client secret needs to be copied during creation, it can't be done afterwards.

-  "Under the section Authorization API add auth for"
it is now called API permissions and it is not clear which api to add the permission to. (Microsoft Graph)

**Comment by hevelius** [comment]
Hey @ddt3 thanks for your message. I'll change the instructions as soon as possible. Anyway every contribute is really appreciated. Feel free to do a pull request.

**Comment by ddt3** [comment]
Pull request created :-) #11
